common/taskScheduler.go

```diff
@@ -0,0 +1,14 @@
+package common
+
+import (
+	"github.com/fiware/VCVerifier/logging"
+	"github.com/procyon-projects/chrono"
+	"time"
+)
+
+func Schedule(task chrono.Task) {
+	_, err := chrono.NewDefaultTaskScheduler().ScheduleAtFixedRate(task, time.Duration(30)*time.Second)
+	if err != nil {
```

Comment:
We are just one tiny log message away from a running, healthy & unconfigured container :D
Think it would be better to hand back the exception.